### DISCUSSION TITLE
Vertical divider?

### DISCUSSION THREAD

**Discussion by SDBonhof** [material-components/material-web#5081] [Q&A]
The [md docs](https://m3.material.io/components/divider/guidelines#77037164-6253-422f-afcd-825eb48922e7) have specifications for a vertical divider. The divider element is marked as stable in this repository, but I don't see a `vertical` option in the docs or in the source code. Is this on the roadmap to be implemented?

Thanks!

**Comment by Paria-Heidari** [comment]
Hi,
I also tried` direction="vertical"` but it doesn't work! 
Just added a class 
```
.divider--vertical {
    width: 1px;
    height: 100%;
    margin: 0 auto;
}
```

Please share the docs or the source code that are related to [md divider doc] (https://m3.material.io/components/divider/guidelines#77037164-6253-422f-afcd-825eb48922e7)

Thanks!

**Reply by asyncliz** [reply]
This isn't a feature that `<md-divider>` supports right now, but I think it's a good candidate for a contribution PR!
